pkg/monitoring/pb/data-feeds/on-chain/registry/feed_updated.proto

```diff
@@ -23,6 +23,8 @@ message FeedUpdated {
   string block_hash = 6;
```

Comment:
Where do we plan to eventually move the `data-feeds`, specifically these `data-feeds.on-chain` protos? (these are common across chains)